### PR TITLE
chore: add cds.wtf site content

### PR DESCRIPTION
**Pull request by coryrylan**
This adds the source of the https://cds.wtf site to the starters repo. Once the stackblitz demos are forked to the clr-angular stackblitz account the links can be updated and the page deployed to netlify. Once live netlify can be pointed to the migrated domain.

This could also be improved on by having a CI job to auto deploy the demo site and possibly auto update between versions. Potential option is to migrate to a clarity subdomain as well like `https://demos.clarity.design` or `https://starters.clarity.design`

### CODE REVIEW

File: site/index.html [hunk 1]
```diff
@@ -0,0 +1,684 @@
...
+      --width: 18px;
+      --height: 18px;
+    }
+  </style>
+  <script async src="https://www.googletagmanager.com/gtag/js?id=G-WM2WWGG0K5"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
+    gtag('js', new Date());
+    gtag('config', 'G-WM2WWGG0K5');
```
Comment (ashleyryan) [inline site/index.html:82]: is this your GA or the team's?

**Review by ashleyryan — APPROVED** [review]
Thank you! 

I was talking to @Jinnie this morning about the possibility of having a demos subdomain and redirecting to there from cds.wtf

File: site/index.html (hunk 1 above)
Comment (coryrylan) [reply site/index.html:82]: Good catch, will remove. The clarity.design domain can be added later

**Comment by coryrylan** [comment]
@ashleyryan  Yeah if a demos.clarity.design subdomain is made for the demos I can setup a permanent redirect to it from cds.wtf. Feel free to ping me if one gets set up, that may be easier long term to maintain vs two custom domains.

**Pull request merged**